<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uirendersceneoptions</class>
 <widget class="QWidget" name="uirendersceneoptions">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scene Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelPathOptions">
         <property name="text">
          <string>Path Options</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="labelAllPaths">
           <property name="text">
            <string>All Paths</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="cbActiveNEEs">
           <property name="text">
            <string>show NEEs</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="dsbLineWidth">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cbNEEs">
           <property name="text">
            <string>show NEEs</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="sliderPathOpacity">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelActivePath">
           <property name="text">
            <string>Active Path</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QSlider" name="sliderActivePathOpacity">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="dsbActiveLineWidth">
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelLineWidth">
           <property name="text">
            <string>Line Width</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelOpacity">
           <property name="text">
            <string>Opacity</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbResetPathOptions">
         <property name="text">
          <string>Reset Path Options</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelSceneOptions">
         <property name="text">
          <string>Scene Options</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QSlider" name="sliderSceneOpacity">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelMotionSpeed">
           <property name="text">
            <string>Camera Speed</string>
           </property>
           <property name="buddy">
            <cstring>sliderCameraSpeed</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cbCameraFocusIntersection">
           <property name="text">
            <string>Focus Selected Intersection</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QSlider" name="sliderCameraSpeed">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelSceneOpacity">
           <property name="text">
            <string>Opacity</string>
           </property>
           <property name="buddy">
            <cstring>sliderSceneOpacity</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbResetSceneOptions">
         <property name="text">
          <string>Reset Scene Options</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label3DDataOptions">
         <property name="text">
          <string>3D Data Options</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelColormap">
           <property name="text">
            <string>Colormap</string>
           </property>
           <property name="buddy">
            <cstring>cbColormap</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbColormap">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelCmapLabel">
           <property name="text">
            <string>Label</string>
           </property>
           <property name="buddy">
            <cstring>leCmapLabel</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leCmapLabel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelCmapMin">
           <property name="text">
            <string>Min:</string>
           </property>
           <property name="buddy">
            <cstring>dsbCmapMin</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsbCmapMin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCmapMax">
           <property name="text">
            <string>Max:</string>
           </property>
           <property name="buddy">
            <cstring>dsbCmapMax</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsbCmapMax">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QCheckBox" name="cbShowColorbar">
           <property name="text">
            <string>show colorbar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="cbShow3DData">
           <property name="text">
            <string>show 3D data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbReset3DDataOptions">
         <property name="text">
          <string>Reset 3D Data Options</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbActiveNEEs</tabstop>
  <tabstop>cbNEEs</tabstop>
  <tabstop>dsbActiveLineWidth</tabstop>
  <tabstop>dsbLineWidth</tabstop>
  <tabstop>sliderActivePathOpacity</tabstop>
  <tabstop>sliderPathOpacity</tabstop>
  <tabstop>pbResetPathOptions</tabstop>
  <tabstop>sliderSceneOpacity</tabstop>
  <tabstop>sliderCameraSpeed</tabstop>
  <tabstop>cbCameraFocusIntersection</tabstop>
  <tabstop>pbResetSceneOptions</tabstop>
  <tabstop>cbColormap</tabstop>
  <tabstop>leCmapLabel</tabstop>
  <tabstop>dsbCmapMin</tabstop>
  <tabstop>dsbCmapMax</tabstop>
  <tabstop>cbShow3DData</tabstop>
  <tabstop>cbShowColorbar</tabstop>
  <tabstop>pbReset3DDataOptions</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
